test/unittests/execute_test.cpp

```diff
@@ -1,21 +1,37 @@
 #include "execute.hpp"
 #include "limits.hpp"
 #include "parser.hpp"
+#include <gmock/gmock.h>
 #include <gtest/gtest.h>
 #include <test/utils/asserts.hpp>
 #include <test/utils/hex.hpp>
 
 using namespace fizzy;
 
+MATCHER(Trapped, "")
+{
+    return arg.trapped == true;
+}
+MATCHER(Result, "")
+{
+    return arg.trapped == false && arg.stack.size() == 0;
+}
+MATCHER_P(Result, value, "")
+{
+    return arg.trapped == false && arg.stack.size() == 1 && arg.stack[0] == uint64_t(value);
+}
+MATCHER_P2(Result, value1, value2, "")
```

Comment:
Where do we need this?

Reply:
There were some tests doing it, but couldn't find them yesterday anymore.